lib/Net/SAML2/IdP.pm

```diff
@@ -66,7 +66,7 @@ has 'cacert'   => (isa => 'Maybe[Str]',   is => 'ro', required => 1);
 has 'sso_urls' => (isa => 'HashRef[Str]', is => 'ro', required => 1);
 has 'slo_urls' => (isa => 'Maybe[HashRef[Str]]', is => 'ro');
 has 'art_urls' => (isa => 'Maybe[HashRef[Str]]', is => 'ro');
-has 'certs'    => (isa => 'HashRef[Str]',        is => 'ro', required => 1);
+has 'certs'    => (isa => 'ArrayRef[HashRef[Str]]', is => 'ro', required => 1);
```

Comment:
Isn't this an `hashref[arrayref[str]]`? multiple signing keys, have `@{$certs->{signing}}`